tests/unit/components/tab/Tab.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import Tab from '~/components/tab/Tab'
+
+describe('Tab', () => {
+  it('renders the label', () => {
+    const label = 'label text'
+    render(<Tab activeTab label={label} onClick={() => {}} />)
+    const tabLabel = screen.getByText(label)
+    expect(tabLabel).toBeInTheDocument()
+  })
+
+  it('applies activeTab if it is true', () => {
+    const label = 'label text'
+    render(<Tab activeTab label={label} onClick={() => {}} />)
+    const tabButton = screen.getByRole('button')
```

Comment:
i think it would be clearer to move this text 'button' in separate value and call something like 'tabButtonRole'

Reply:
i decided to leave as it is